dojo/finding/views.py

```diff
@@ -818,6 +822,30 @@ def edit_finding(request, fid):
                         status.last_modified = timezone.now()
                         status.save()
 
+            if system_settings.false_positive_history:
+                # If the finding is being marked as a false positive we dont need to call the
+                # fp history function because it will be called by the save function
+
+                # If finding was a false positive and is being reactivated: retroactively reactivates all equal findings
+                if old_finding.false_p and not new_finding.false_p:
+                    if system_settings.retroactive_false_positive_history:
+                        logger.debug('FALSE_POSITIVE_HISTORY: Reactivating existing findings based on: %s', new_finding)
+
+                        existing_fp_findings = match_finding_to_existing_findings(
+                            new_finding, product=new_finding.test.engagement.product
+                        ).filter(false_p=True)
+
+                        for fp in existing_fp_findings:
+                            logger.debug('FALSE_POSITIVE_HISTORY: Reactivating false positive %i: %s', fp.id, fp)
+                            fp.active = new_finding.active
+                            fp.verified = new_finding.verified
+                            fp.false_p = False
+                            fp.out_of_scope = new_finding.out_of_scope
+                            fp.is_mitigated = new_finding.is_mitigated
+                            fp.last_reviewed = timezone.now()
+                            fp.last_reviewed_by = request.user
```

Comment:
This might get a little confusing as it would create instances of findings being "reviewed" by someone who may not have any knowledge of the findings existence